### ISSUE TITLE
Scaling Issues in .bcf files and beyond

### ISSUE THREAD

**Issue by k8macarthur** [hyperspy/hyperspy#1726]
The .bcf import function automatically imports the spectrum image with the scale in micro. These leads to two problems:

- [ ] The scale bar appears in the ridiculous scale. Could this be forced to be greater than 1 for example?

- [ ] The EDX quantification function assumes the pixel size is in units of nm but doesn't actually check for this. Therefore a check for units needs to be added.

 I would also suggest that we could added a check to the import function to automatically convert the scale to smaller units if necessary?

**Comment by francisco-dlp** [comment]
Good points. @ericpre did some work on this using pint but I can't remember how far he got into solving the issue.

**Comment by k8macarthur** [comment]
This came up during the workshop. In the short term, fixing the quantification function should be relatively easy. I'll try and get to that asap. I think @magnunor  had some thoughts about how to update the scale bar function.

**Comment by magnunor** [comment]
I've been thinking about updating the scale bar function, as mentioned in https://github.com/hyperspy/hyperspy/issues/1066. But I guess the scaling issue discussed here is a separate thing, since https://github.com/hyperspy/hyperspy/issues/1066 only deals with the visualization, not the numbers or text.

**Comment by sem-geologist** [comment]
at this moment it is in µm as:
1. bcf internally are in µm
2. µm is universal unit in between cm and nm orders.
E.g. in geology (which was my main aim) lots of mapping are done in cm scale (with image width up to 2 cm) nm scale would make no sense, µm is just right.

The best would be to cast all units into SI and then use some library to show appropriate order.
The moment I wrote the library I saw that all io-pluggins returned hard coded units.
